pipeline.ipynb

```diff
@@ -1089,7 +1199,7 @@
     "    bad_units.sort()\n",
     "    if len(bad_units)>0:\n",
     "        hv_res = (hv.NdLayout({\n",
-    "            \"Spatial Footprin\": regrid(hv.Dataset(A_spatial.sel(unit_id=bad_units).rename('A'))\n",
+    "            \"Spatial Footprin\": regrid(hv.Dataset(A_spatial.compute().sel(unit_id=bad_units).rename('A'))\n",
```

Comment:
separate commit/PR?

Reply:
yes, see below, I've reverted the pipeline.ipynb notebook